saleor/webhook/payloads.py

```diff
@@ -996,3 +999,83 @@ def generate_translation_payload(
         translation_data.update(context)
 
     return json.dumps(translation_data)
+
+
+MAX_PAYLOAD_ATTRIBUTE_SIZE = 1024 * 125
+
+
+def truncate_str_to_byte_limit(
+    text: Optional[str], limit=MAX_PAYLOAD_ATTRIBUTE_SIZE, encoding="utf-8"
+) -> Optional[str]:
+    if not text:
+        return text
+    return text.encode(encoding)[:limit].decode(encoding, "ignore")
+
+
+def generate_api_call_payload(request, response):
+    content_length = int(request.headers.get("Content-Length", 0))
+    response_body = response.content.decode(response.charset)
+    request_body = None
+    if request.POST:
+        request_body = json.dumps(dict(request.POST))
```

Comment:
Can' we use here `response.json()`?

Reply:
The idea was to be universal and get data to report even if returned content isn't proper JSON. We later send and process it as a string, so there is no need to JSON parse it.